app/src/main/java/pm/gnosis/heimdall/ui/splash/SplashActivity.kt

```diff
@@ -43,6 +44,7 @@ class SplashActivity : BaseActivity() {
     private fun handleAction(action: ViewAction) {
         when (action) {
             is StartMain -> startMain()
+            is StartUnlock -> startUnlock()
```

Comment:
If the action is `StartUnlock` then after unlocking it goes back to this activity (Splash)

Reply:
Yes, it will then check again and start main if it was unlocked

Reply:
👌